Updated playbooks

cs-rohit-khune opened the pull request:
Unit Test Cases

Mantis #0857352
- Validated that the updated jinja code to check IP address reputation is working correctly in playbooks `Investigate Brute Force Attempt (FortiSIEM)` and `Investigate Brute Force Attempt`

cs-rohit-khune committed:
Updated playbooks

Unit Test Cases

Mantis #0857352
- Validated that the updated jinja code to check IP address reputation is working correctly in playbooks `Investigate Brute Force Attempt (FortiSIEM)` and `Investigate Brute Force Attempt`

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,7 +1,4 @@
-    - A new tag **Brute Force Attack** &ndash; helps with filtering from the left pane when using the **Search By Tags** option
-- The description of the scenario **Brute Force Attempt (FortiSIEM)** is now more comprehensive
+- Modified `Investigate Brute Force Attempt (FortiSIEM)` and `Investigate Brute Force Attempt` playbooks
+    - Updated jinja logic to check malicious IP Address
```
This description is not enough. Please add more details

cs-rohit-khune replied:
Updated the release notes, please check

cs-suyog-jinturkar reviewed (commented):
Added review comments

cs-rohit-khune committed:
Updated Demo Record and Playbooks

- Updated demo record Source IP address to get valid malicious result
- Updated mock data format for VT to the latest
- Validated the playbooks `Investigate Brute Force Attempt (FortiSIEM)` and `Investigate Brute Force Attempt` are running correctly for updated jinja and for VT connector version 2.x onwards
